src/SSHDebugPS/Docker/DockerConnection.cs

```diff
@@ -243,10 +245,14 @@ private ICommandRunner GetCommandRunner(DockerContainerTransportSettings setting
                 return new RemoteCommandRunner(settings, OuterConnection);
         }
 
+        private static readonly string IncorrectContainerType = "IncorrectContainerType";
         protected override string ProcFSErrorMessage
         {
             get
             {
+                HostTelemetry.SendEvent(Telemetry.Event_DockerConnectionTypesNotMatching, new KeyValuePair<string, object>[] {
```

Comment:
So I want to call out that "wrong container type" isn't the only way ProcFS fails, but just one of them so this telemetry point is misleading. It might be better to change the telemetry to just call it procfs fails.